command/kv_metadata_put.go

```diff
@@ -71,14 +71,13 @@ func (c *KVMetadataPutCommand) Flags() *FlagSets {
 	f.IntVar(&IntVar{
 		Name:    "max-versions",
 		Target:  &c.flagMaxVersions,
-		Default: 0,
+		Default: -1,
 		Usage:   `The number of versions to keep. If not set, the backend’s configured max version is used.`,
 	})
 
-	f.BoolVar(&BoolVar{
```

Comment:
It seems like `BoolVar` is no longer being used anywhere. Could we clean up those struct definitions?

Reply:
I don't see any reference to `BoolVar` in [command/kv_metadata_put.go](https://github.com/hashicorp/vault/blob/c31ccf7fcf55fd81c48e497452d30f808f9a3038/command/kv_metadata_put.go).

Reply:
Right, but the `BoolVar` struct definition still exists. I think it can just be removed entirely, right?

Reply:
Nevermind, I think I was hunting for references in the wrong repo!